guides/common/modules/proc_assigning-ansible-roles-to-an-existing-host.adoc

```diff
@@ -21,7 +21,7 @@ Use the *<* button to remove a role, or the *<<* button to remove all roles.
 . Click *Confirm*.
 
 After you assign Ansible roles to hosts, you can use Ansible for remote execution.
-For more information, see {ConfiguringAnsibleDocURL}ssh-keys-for-remote-execution-overview_ansible[Distributing SSH Keys for Remote Execution].
+For more information, see https://docs.theforeman.org/nightly/Configuring_Ansible/index-foreman-el.html#ssh-keys-for-remote-execution-overview_ansible[Distributing SSH Keys for Remote Execution].
```

Comment:
Please revert this change.

Reply:
Hi, @maximiliankolb !  The original links weren't passing the PR checks, and when I changed them, the checks all passed.  Any idea why this might be the case?

Reply:
That sounds strange, yes. But because the pipeline for "master" branch build, your feature branch without any change in this file and without change to the anchor itself should definitely work!